emrun.py

```diff
@@ -1820,6 +1823,10 @@ def run(cmd):
     else:
       browser_stderr_handle = open(options.log_stderr, 'a')
   if options.run_browser:
+    if options.browser_url:
+        url = options.browser_url
+        logv('Connecting browser to URL ' + url)
+        browser += browser_args + [url]
```

Comment:
This block looks misplaced: the line `browser += browser_args + [url]` already occurs on line 1771 (1768 in old code) above: https://github.com/Unity-Technologies/emunittest/pull/10/files#diff-dc0b4ef45b945dfeb75f7462aacf00470588c37739844b995ba7ecf4c895bd78L1768

So essentially this would cause two URLs to be passed on the command line I think.

Can you remove this change, and move it back up around line 1771 so it will read

```py
      # In Windows cmdline, & character delimits multiple commmands, so must
      # use ^ to escape them.
      if browser_exe == 'cmd':
        url = url.replace('&', '^&')
      url = url.replace('0.0.0.0', 'localhost')
      if options.browser_url:
        url = options.browser_url
        logv('Connecting browser to URL ' + url)
      browser += browser_args + [url]
```
This way the `url` added to `browser` cmdline will be replaced before first adding it to the cmdline list.

Reply:
Hmm, changing it to the above actually gave me the following:
`http://localhost:6931/?--browser-url&http://localhost:8000/`

I can dig into it more later though, its probably a simple change

Reply:
I wonder if there was some typo?  What did the verbose log say?

Reply:
Yep looks like I must have made a mistake, should be fixed now